lib/watch.js

```diff
@@ -383,18 +376,15 @@ function createExtraFilesWatcher(options, app, runReview, onError, request) {
   return chokidar
     .watch(
       request.files.map(
-        ({pattern, included}) =>
-          (included ? '' : '!') + OsHelpers.makePathOsAgnostic(pattern)
+        ({pattern, included}) => (included ? '' : '!') + pattern
```

Comment:
Does chokidar support Windows-style paths?

Reply:
~~No idea. I should check.~~ They normalize it (at least they used to, we're still on v3, so they still should?), so you can use either, but it _should_ return a POSIX path rather than a DOS path.
I had a chat with my dad about the elm-review's handling of drives, which basically ended with `makePathOsAgnostic` uses `/.:/` to strip the drive, when elm-review should keep the drive so that you can execute it across drives (which MS wants to make more common). `pathe` handles proper agnosticism, so the pattern passed in should be correct, but I should also just check that as well.